runtime/env-defaults

```diff
@@ -18,3 +18,7 @@ DB_USER=${DB_USER:-}
 DB_PASS=${DB_PASS:-}
 
 DB_UNACCENT=${DB_UNACCENT:-false}
+
+DB_POSTGIS=${DB_POSTGIS:-false}
+DB_POSTGIS_HSTORE=${DB_POSTGIS_HSTORE:-true}
+DB_POSTGIS_TOPOLOGY=${DB_POSTGIS_TOPOLOGY:-true}
```

Comment:
Hi Sameer,

sorry for sloppiness on my part. I don't know how to fix that in this pull request, so I have created [another one](https://github.com/sameersbn/docker-postgresql/pull/47).

I decided to drop the hstore extension, as I it looks like it's not a PostGIS specific extension.